bucket/yt-dlp.json

```diff
@@ -18,6 +17,12 @@
         }
     },
     "bin": "yt-dlp.exe",
+    "pre_install": [
+        "$manifest.persist | ForEach-Object {",
+        "    if (-not (Test-Path \"$persist_dir\\$_\")) { New-Item \"$dir\\$_\" -ItemType file | Out-Null }",
+        "}"
+    ],
```

Comment:
This was copied from `everything.json` to prevent scoop from creating a directory called `yt-dlp.conf`. I don't know if there is another proper way to do this